test/integration/analytics_int_test.go

```diff
@@ -377,6 +377,45 @@ func (suite *AnalyticsIntegrationTestSuite) TestInputError() {
 	}
 }
 
+func (suite *AnalyticsIntegrationTestSuite) TestAttempts() {
+	suite.OnlyRunForTags(tagsuite.Analytics)
+
+	ts := e2e.New(suite.T(), true)
+	defer ts.Close()
+
+	asyData := strings.TrimSpace(`project: https://platform.activestate.com/ActiveState-CLI/test?commitID=9090c128-e948-4388-8f7f-96e2c1e00d98`)
+	ts.PrepareActiveStateYAML(asyData)
+
+	cp := ts.SpawnWithOpts(
+		e2e.WithArgs("activate", "ActiveState-CLI/Alternate-Python"),
+		e2e.AppendEnv(constants.DisableRuntime+"=false"),
+		e2e.WithWorkDirectory(ts.Dirs.Work),
+	)
+
+	cp.Expect("Creating a Virtual Environment")
+	cp.Expect("Activated")
+	cp.WaitForInput(120 * time.Second)
+
+	cp.SendLine("python3 --version")
+	cp.Expect("Python 3.")
+
+	time.Sleep(time.Second) // Ensure state-svc has time to report events
+
+	suite.eventsfile = filepath.Join(ts.Dirs.Config, reporters.TestReportFilename)
+	events := parseAnalyticsEvents(suite, ts)
+
+	var foundAttempts int
+	for _, e := range events {
+		if strings.Contains(e.Category, "runtime") && strings.Contains(e.Action, "attempt") {
+			foundAttempts++
+		}
+	}
+
+	if foundAttempts < 2 {
```

Comment:
Unlike with heartbeats we should be able to get a consistent number here, so please assert `== 2`.